tools/ms/libusbx_setup.cmd

```diff
@@ -2,12 +2,12 @@
 setlocal EnableDelayedExpansion
 
 if [%1]==[] goto usage
-
+if exist %SDL_LIBUSB_DIR% goto msg
 set BUILD_TYPE=%1
 set LIBS_DIR="%SDL_ROOT_DIR%\src\3rd_party-static"
 set LIBUSBX_SRC_DIR_NAME=libusbx-1.0.17
 set LIBUSBX_SRC_DIR=%LIBS_DIR%\%LIBUSBX_SRC_DIR_NAME%
-set LIBUSBX_BUILD_DIR=%SDL_BUILD_DIR%\libusbx_win_%BUILD_TYPE%
+set LIBUSBX_BUILD_DIR=%SDL_LIBUSB_DIR%\libusbx_win_%BUILD_TYPE%
```

Comment:
@antongovorov my question is still here:
where you are going to place openssl build in case `SDL_LIBUSB_DIR` is not set or empty?